networktables.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
remove this file. Please make sure not to commit files that aren't your code. This file was autogenerated by some tool.